e2e/cypress/integration/messaging/message_by_aeroplane_icon_spec.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('M18677 - Clicking on airplane icon does not open file attachment modal but sends the message', () => {
+    before(() => {
+        // # Login as "user-1" and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+
+        // # resize window to mobile view
+        cy.viewport('iphone-6');
+    });
+
+    it('Message should send, File attachment modal should not open', () => {
+        // # type some characters in the message box
+        const message = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. ';
+        cy.get('#post_textbox').type(message);
+
+        // # click send
+        cy.get('.send-button').click();
+
+        // # check if message sent correctly
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#postMessageText_${postId}`).should('be.visible').and('have.text', message);
+        });
+
+        // # attachment modal does not show up
+        cy.get('.a11y__popup ').should('not.exist');
```

Comment:
Does this represent file attachment modal? I've checked it and looks like not; though I don't know if it's possible to check.  I'm fine with this as this could be like checking when modal like emoji picker (closer to paper-plane icon) is hidden/closed.

Reply:
these are all the classes the <ul> element "a11y__popup Menu dropdown-menu MenuWrapperAnimation-enter-done" .  what is the best way to check for it?